### ISSUE TITLE
Maybe wrong result

### ISSUE THREAD

**Issue by nikitadol** [D00mch/DartClojure#17]
```logs
> npx dartclojure "a > b || (c > d && d > e && e > f)" -m "m"
(or (> a b) (> c d e f))
> npx dartclojure "a > b || (c > d && d > e && e < f)" -m "m"
(or (> a b) (> c d e f))

```

**Comment by D00mch** [comment]
Why do you think it's wrong?

**Comment by nikitadol** [comment]
Unfortunately I have a problem with the test

This code:
```dart
void main() {
  final a = 1;
  final b = 1;
  final c = 4;
  final d = 3;
  final e = 2;
  final f = 1;
  print(a > b || (c > d && d > e && e > f)); // true
  print(a > b || (c > d && d > e && e < f)); // false
}
```

I pasted here:
```command
> npx dartclojure "void main() {                                                        
  final a = 1;
  final b = 1;
  final c = 4;
  final d = 3;
  final e = 2;
  final f = 1;
  print(a > b || (c > d && d > e && e > f));
  print(a > b || (c > d && d > e && e < f));
}" -m "m"
```
And got this code:
```clojure
(defn main
  []
  (def a 1)
  (def b 1)
  (def c 4)
  (def d 3)
  (def e 2)
  (def f 1)
  (dart:core/print (or (> a b) (> c d e f)))
  (dart:core/print (or (> a b) (> c d e f))))
```

But it doesn't compile (with `clj -M:cljd compile`):
```command
Execution error (AssertionError) at cljd.compiler/write (compiler.cljc:3822).
Assert failed: (= (count args) 1)
```

**Comment by D00mch** [comment]
I see. That's a correct (not idiomatic though) Clojure code, I can run it with Clojure (java host):
<img width="720" alt="image" src="https://user-images.githubusercontent.com/14236531/207969685-299b98c4-c680-4b03-9f6a-41383f79558b.png">

I will ask the cljd devs if they know about it, and will be back here.

**Comment by D00mch** [comment]
Seems like a bug. I found that 3 args work:
```
(defn main []
  (defn main
    []
    (def a 1)
    (def b 1)
    (def c 4)
    (def d 3)
    (def e 2)
    (def f 1)
    (dart:core/print (or (> a b) (> c d e)))
    (dart:core/print (or (> a b) (> c d e)))))
```

**Comment by nikitadol** [comment]
> <img width="720" alt="image" src="https://user-images.githubusercontent.com/14236531/207969685-299b98c4-c680-4b03-9f6a-41383f79558b.png">

Judging by your screenshot, the code prints true in both cases - Dart prints true false

**Comment by D00mch** [comment]
Oh, I finally see this time :) 

The correct code should be like: 
`(or (> a b) (and (> c d e) (< e f))`
or
`(or (> a b) (> c d f e))`.

**Comment by D00mch** [comment]
Translation bug is fixed in 0.2.13 ([clojars](https://clojars.org/org.clojars.liverm0r/dartclojure), [npm](https://www.npmjs.com/package/dartclojure)).

Compilation issue is fixed here https://github.com/Tensegritics/ClojureDart/commit/b482135d92c830fbf7f2fefab2f48803ffd032d6.
